README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">Grüvee</h1>
 <p align="center">
-  <strong>Get Grüvee with a new social, collabortive playlist for iPhone and Android</strong>
+  <strong>Get Grüvee with a new social, collaborative playlist for iPhone and Android</strong>
```

Comment:
Thanks for catching the spelling mistake! Can you please change this entire line to say:

"Let's get Grüvee with a new social, collaborative playlist for iPhone and Android"